package.json

```diff
@@ -117,52 +110,52 @@
     "@types/lodash": "^4.14.123",
     "@types/react": "16.8.8",
     "@types/rosie": "^0.0.36",
-    "@types/storybook-readme": "^4.0.0",
+    "@types/storybook-readme": "^5.0.1",
     "@types/storybook__addon-actions": "^3.4.2",
     "@types/storybook__addon-info": "^4.1.1",
     "@types/storybook__react": "^4.0.1",
+    "antd": "^3.0.0",
     "auto-changelog": "^1.12.0",
     "awesome-typescript-loader": "^5.2.1",
     "babel-loader": "^8.0.5",
     "babelrc-rollup": "^3.0.0",
     "coveralls": "^3.0.3",
     "css-loader": "^2.1.1",
     "enzyme": "^3.9.0",
-    "enzyme-adapter-react-16": "^1.11.2",
+    "enzyme-adapter-react-16": "^1.12.1",
     "faker": "^4.1.0",
     "gh-pages": "^2.0.1",
-    "jest": "^24.5.0",
-    "jest-environment-jsdom": "^24.5.0",
+    "jest": "^24.7.1",
+    "jest-environment-jsdom": "^24.7.1",
+    "lodash": "^4.0.0",
+    "mobx": "^5.0.0",
+    "mobx-react": "^5.0.0",
+    "moment": "^2.0.0",
     "np": "^4.0.2",
+    "react": "^16.0.0",
     "react-docgen-typescript-webpack-plugin": "^1.1.0",
-    "react-dom": "^16.8.4",
+    "react-dom": "^16.8.6",
     "regenerator-runtime": "^0.13.2",
-    "rollup": "^1.7.0",
+    "rollup": "^1.9.0",
     "rollup-plugin-babel": "^4.3.2",
-    "rollup-plugin-commonjs": "^9.2.1",
-    "rollup-plugin-node-resolve": "^4.0.1",
+    "rollup-plugin-commonjs": "^9.3.4",
+    "rollup-plugin-node-resolve": "^4.2.1",
     "rollup-plugin-peer-deps-external": "^2.2.0",
     "rosie": "^2.0.1",
     "source-map-explorer": "^1.8.0",
     "storybook-readme": "^5.0.1",
     "style-loader": "^0.23.1",
-    "tslint": "^5.14.0",
-    "typescript": "^3.3.4000"
+    "tslint": "5.14.0",
+    "typescript": "3.3.4000"
   },
   "dependencies": {
-    "@mighty-justice/smart-bool": "^0",
-    "@mighty-justice/utils": "^1",
-    "antd": "^3",
-    "class-autobind-decorator": "^3",
-    "classnames": "^2",
-    "date-fns": "^1",
-    "flat": "^4",
-    "http-status-codes": "^1",
-    "iso8601-duration": "^1",
-    "lodash": "^4",
-    "mobx": "^5",
-    "mobx-react": "^5",
-    "moment": "^2",
-    "react": "^16"
+    "@mighty-justice/smart-bool": "1.0.0",
+    "@mighty-justice/utils": "1.2.1",
+    "class-autobind-decorator": "3.0.1",
+    "classnames": "2.2.6",
+    "date-fns": "1.30.1",
+    "flat": "4.1.0",
+    "http-status-codes": "1.3.2",
+    "iso8601-duration": "1.2.0"
```

Comment:
Why the restrictions for these packages? I understand for packages like `tslint` and `typescript`

Reply:
Good point